src/components/modeler/moveWithArrowKeys.js

```diff
@@ -0,0 +1,49 @@
+import store from '@/store';
+import PaperManager from '@/components/paperManager';
+
+const movementKeys = {
+  up: ['Up', 'ArrowUp'],
+  down: ['Down', 'ArrowDown'],
+  left: ['Left', 'ArrowLeft'],
+  right: ['Right', 'ArrowRight'],
+};
+
+export default function setUpArrowKeyEventHandlers() {
+  document.addEventListener('keydown', event => {
+    const highlightedShape = store.getters.nodeShape(store.getters.highlightedNode);
+    if (!highlightedShape) {
+      return;
+    }
+
+    if (!isArrowKeyPressed(event.key)) {
+      return;
+    }
```

Comment:
How would you feel about combining these if statement like:
`if (!highlightedShape || !isArrowKeyPressed(event.key))`